src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/ConsentShutter.cshtml

```diff
@@ -0,0 +1,28 @@
+﻿@page
+@model FamilyHubs.Referral.Web.Pages.ProfessionalReferral.ConsentShutterModel
+@{
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <form method="post">
+            <fieldset class="govuk-fieldset">
```

Comment:
Using a fieldset here isn't appropriate (I know the prototype has one, but this isn't a form).

We should remove the fieldset, legend etc. and use appropriate classes for elements such as the h1.

Reply:
Removed